app/services/RegistrationService.scala

```diff
@@ -126,7 +126,7 @@ class RegistrationService @Inject()(
   private def getExclusionDetailsForTransferringMSID(answers: UserAnswers): EtmpExclusionDetails = {
     val country = answers.get(EuCountryPage).getOrElse(throw new Exception("No country provided"))
     val moveDate = answers.get(MoveDatePage).getOrElse(throw new Exception("No move date provided"))
-    val taxNumber = answers.get(TaxNumberPage).getOrElse(throw new Exception("No tax number provided"))
+    val euVatNumber = answers.get(EuVatNumberPage).getOrElse(throw new Exception("No tax number provided"))
```

Comment:
Sorry for changing my review, but in registrations when we send to core validation or to registration endpoint, we strip out the country code - can you please add the similar functionality of removing the country code from the vat number before sending? Thanks

Reply:
Will do, thanks

Reply:
Now using `convertedVatNumber` below